src/app/loadout-builder/generated-sets/SetStats.m.scss

```diff
@@ -1,25 +1,30 @@
 @use '../../variables' as *;
 
 .container {
+  color: #ddd;
   font-size: 14px;
-  flex-wrap: wrap;
+  display: flex;
+  flex-flow: row wrap;
+  gap: 4px 8px;
+  align-items: center;
 }
 
 .statIcon {
   vertical-align: middle;
   height: 16px;
   width: 16px;
+  font-size: 16px;
+  margin-right: 1px;
 }
 
 .light {
   color: $gold;
-  margin-right: 14px;
   white-space: nowrap;
 
   :global(.app-icon) {
-    font-size: 10px;
-    vertical-align: -0.125em;
-    margin-right: 0;
+    vertical-align: 4px;
+    font-size: 8px;
+    margin-right: 2px;
```

Comment:
```suggestion
    vertical-align: 3px;
    font-size: 8px;
    margin-right: 1px;
```
This brings the Power 'star' a smidge closer to the text, which is a bit more like how it's rendered in-game.

Before:
![image](https://github.com/DestinyItemManager/DIM/assets/17512262/d3e481c5-6f00-4272-8035-e9086e542c99)

After:
![image](https://github.com/DestinyItemManager/DIM/assets/17512262/0e9fda05-e324-453b-9956-ea9969a330a2)